tests/e2e_tests/client_utils.py

```
@@ -219,27 +219,27 @@ def search_points(self, collection_name: str) -> Dict[str, Any]:
     def create_snapshot(self, collection_name: str = "test_collection") -> Optional[str]:
         """Create a snapshot of the collection."""
         snapshot_info = self.client.create_snapshot(collection_name=collection_name)
         return snapshot_info.name if snapshot_info else None
-    
+
     def download_snapshot(self, collection_name: str, snapshot_name: str) -> Tuple[bytes, str]:
         """Download a snapshot and return its content and checksum."""
         # Note: qdrant-client doesn't have a direct method to download snapshot content
         # This would need to be implemented using the REST API directly
         snapshot_url = f"http://{self.host}:{self.port}/collections/{collection_name}/snapshots/{snapshot_name}"
         response = requests.get(snapshot_url)
```

Ruff (0.14.1)

Probable use of `requests` call without timeout

(S113)